addons/pos_self_order/__init__.py

```diff
@@ -3,3 +3,7 @@
 from . import controllers
 from . import models
 from . import tests
+
+def _post_self_order_post_init(env):
+    sessions = env['pos.session'].search([('state', '!=', 'closed')])
+    env['pos.session']._create_pos_self_sessions_sequence(sessions)
```

Comment:
Is it possible to verify if there's sessions before call `_create_pos_self_sessions_sequence` ?